cloud/google/machineactuator.go

```diff
@@ -256,10 +256,15 @@ func (gce *GCEClient) Create(cluster *clusterv1.Cluster, machine *clusterv1.Mach
 		if gce.machineClient == nil {
 			labels[BootstrapLabelKey] = "true"
 		}
+		tags := []string{"https-server"}
+		if !util.IsMaster(machine) {
+			tags = append(tags, fmt.Sprintf("%s-worker", cluster.Name))
```

Comment:
What uses this %s-worker tag?

Reply:
Both nodes and masters get tags applied via GCE. They both get the https-server tag but only nodes get %s-worker. The master startup script creates a cloud-config file where it specifies the node-tags as %s-worker, but that doesn't add the tag to the master. I think the cloud-config is just for GCP to know how to create the firewall rules and maybe do some other networking things. I'm not 100% sure.

Reply:
I'll add the tag to the master. This change will be moved to a new PR though.